### ISSUE TITLE
Confussed

### ISSUE THREAD

**Issue by Matthuffy** [fireice-uk/xmr-stak#1737]
why don't you simply ask donate, I would have donated a few k but I saw that you force 2% f hash rate, it is done on purpose hoping that nobody would change it. that is very lame. I now cannot donate for that reason.

**Comment by Matthuffy** [comment]
yes you say, but it is truly hoping nobody sees and changes. or you could ask for donations and we would. sorry you lost my donation for that practice and I wont now use this miner as god kows what else did and no time or effort to look

**Comment by Matthuffy** [comment]
simply delete

**Comment by Spudz76** [comment]
the 2% is very clearly defined in the docs and in the startup messages when running the program

simply compile from source, the code for donation percentage is the only thing in this really obviously named file [here](https://github.com/fireice-uk/xmr-stak/blob/master/xmrstak/donate-level.hpp#L18) change 2.0 to 0.0 - pretty easy work, considering it buys you your ticket on freeloader express forever, and we did the opposite of hiding it - it's the ONLY THING in a file called DONATE-LEVEL.  brain twister, I know

sorry you want free usage of a free program without having to do *any* work at all.  how did you manage to unzip the release and configure it, are you out of breath or in heatstroke???

donation is forced upon the lazy, which sounds totally fair to me
all the compilers needed are also free
if obligatory donation worked, there would be no need for taxes, but we all have taxation right?

secondary reasoning is the more users that compile, the more users that have a compiler environment available to test new unreleased code and help kill bugs.  this is why there will never be a command line option to set donations (too easy) it will always be tied to compiling.  a million users downloading a Release package and just running it does little to no good for figuring out the cause of bugs.

**Comment by Matthuffy** [comment]
Wow careful how you talk. Read again

**Comment by Spudz76** [comment]
read what again?

your story about how we missed out on a quantum-theory donation (that wasn't ever going to really exist anyway) just because you're sour about mining for us 72 seconds every hour, whenever you are too lazy to compile from source? because I already read that one several times before while writing my previous retort

the default donation is completely not predatory, nor hidden, nor with bad intention, it is a convenience fee for using the simplest route (download, unpack, run) which functions as an incentive to bother

your whining about it used more effort than just compiling and moving on, and was 101% only to announce your altruistic nature and berate all the devs and tell them you think their work isn't worth 72 seconds for the other 3528 seconds that were all your profit.

"I would have tipped you 900 dollars if it wasn't for the gratuity-included" tell that to some waitstaff and see if you ever get clean food again

**Comment by Matthuffy** [comment]
Still talking gibberish.

**Comment by Spudz76** [comment]
Then write something other than garbage that results in gibberish.

Fee is 2%, compile or *GO AWAY* are your options, and I thought you had already chosen option 2 so why are you still here causing problems?

**Comment by Matthuffy** [comment]
not true,just remove that 2% garbage. but you are right i found one that works much better, much faster and do not attempt to force you to share hash. so I donated 2000 bucks to them.

Pays to be an honest developer.
